cmd/minishift/cmd/stop.go

```diff
@@ -48,8 +48,11 @@ func runStop(cmd *cobra.Command, args []string) {
 		cluster.RegistrationParameters.SkipUnRegistration = true
 	}
 
+	if err := cluster.UnRegister(api); err != nil {
```

Comment:
Is this not a bit confusing now? First you do a check for skip registration and set a flag and then you still call _cluster.UnRegister(api)_. Should we not just move the UnRegister call into the if statement. This makes cluster.RegistrationParameters.SkipUnRegistration obsolete.